lib/cb/models/implementations/job.rb

```diff
@@ -181,6 +182,9 @@ def set_model_properties
         @zip                          = args['LocationPostalCode'] || ''
         @posting_date                 = args['PostingDate'] || ''
 
+        # JC Custom Fields related
+        @jc_custom_fields             = args['JCCustomField'] || ''
```

Comment:
Could you please give me a sample of this unit test ? I searched before, but found nothing for this case. Thanks so much